client/components/thank-you/style.scss

```diff
@@ -59,3 +41,128 @@
 .thank-you__help-text {
 	margin-bottom: 0.5em;
 }
+
+.thank-you {
+
+	&__container {
+
+		margin-top: calc( -79px + var( --masterbar-height ) );
+
+		@include breakpoint-deprecated( '<660px' ) {
+			margin-top: -1px;
+		}
+	}
+
+	&__header {
+		&-title {
+			line-height: 40px;
+		}
+
+		&-subtitle {
+			padding-top: 4px;
+			margin-bottom: 0;
+		}
+	}
+}
+
+.thank-you__container {
+
+	margin-top: calc( -79px + var( --masterbar-height ) );
+
+	@include breakpoint-deprecated( '<960px' ) {
+		margin-top: calc( -71px + var( --masterbar-height ) );
+	}
+
+	@include breakpoint-deprecated( '<660px' ) {
+		margin-top: -1px;
+	}
```

Comment:
Not for this PR, but the `-79px`/`-71px`/`-1px` magic numbers along with the respective breakpoints definitely need to be shifted to a variable.

OK - I created #56654 for this. We have multiple places in Calypso that are doing weird things with these hard-coded values and breakpoints, and they are all liable to break in future.